src/main/resources/schemas/et_fortnightly_press_list.json

```diff
@@ -0,0 +1,434 @@
+                        "type": "object",
+                        "required": ["sittings"],
+                        "properties": {
+                          "sessionChannel": {
+                            "$id": "#root/courtLists/courtRoom/session/sessionChannel",
+                            "type": "array",
+                            "items": {
+                              "description": "Requested to ensure we host the type of hearing channel in the session",
+                              "title": "Session Channel",
+                              "type": "string",
+                              "examples": [
+                                "VIDEO HEARING"
+                              ],
+                              "default": "",
+                              "pattern": "^.*$"
+                            }
+                          },
+                          "sittings": {
+                            "$id": "#root/courtLists/courtList/courtRoom/session/sittings",
+                            "title": "Sittings",
+                            "type": "array",
+                            "default": [],
+                            "items":{
+                              "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items",
+                              "type": "object",
+                              "required": ["sittingStart", "sittingEnd", "hearing"],
+                              "properties": {
+                                "sittingStart": {
+                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/sittingStart",
+                                  "description": "Sitting Start Time",
+                                  "title": "Sitting Start",
+                                  "type": "string",
+                                  "default": "",
+                                  "examples": []
+                                },
+                                "sittingEnd": {
+                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/sittingEnd",
+                                  "description": "Sitting End Time",
+                                  "title": "Sitting End",
+                                  "type": "string",
+                                  "default": "",
+                                  "examples": []
+                                },
+                                "hearing": {
+                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing",
+                                  "type": "array",
+                                  "items": {
+                                    "type": "object",
+                                    "description": "Hearing details of the case and associated details for the sitting",
+                                    "title": "Hearing",
+                                    "required": ["case"],
```

Comment:
My comment can be entirely solved by adding "hearingType" to this line.

Reply:
Following discussion, flagged this in the ticket. Hearing Type is marked as mandatory on the ticket, but not in the attachments.